src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,5 +1,8 @@
+
+<tiny-task-search #search></tiny-task-search>
```

Comment:
the search component should have been added to the app.component and the logic programed to make the search/filter work that way. Also careful with adding extra empty line